formatter.go

```diff
@@ -157,7 +157,6 @@ func (p *printer) printValue(v reflect.Value, showType, quote bool) {
 			}
 			p.visited[vis] = p.depth
 		}
-
```

Comment:
Nit: deleting this blank line seems unrelated to the substance of the change we're trying to make. Do you mind leaving it in?

Reply:
this was from `gofmt -w -l -s`